client/src/components/add-employer-component/company-info-component/index.jsx

```diff
@@ -0,0 +1,442 @@
+              />
+              <TextField
+                fullWidth
+                sx={{ m: 1, width: "96%" }}
+                id={`secondaryCity${lead.id}`}
+                value={lead.secondaryCity}
+                onChange={(input) =>
+                  handleInputChange(
+                    input.target.value,
+                    lead.id,
+                    "secondaryCity",
+                  )
+                }
+                label="City"
+              />
+              <FormControl fullWidth sx={{ m: 1, width: "47%" }}>
+                <InputLabel id={`secondaryProvinceLabel${lead.id}`}>
+                  Secondary Province
+                </InputLabel>
+                <Select
+                  sx={{ textAlign: "left" }}
+                  labelId={`secondaryProvinceLabel${lead.id}`}
+                  label="Secondary Province"
+                  id={`secondaryProvince${lead.id}`}
+                  name={`secondaryProvince${lead.id}`}
+                  value={lead.secondaryProvince}
+                  onChange={(input) =>
+                    handleInputChange(
+                      input.target.value,
+                      lead.id,
+                      "secondaryProvince",
+                    )
+                  }
+                >
+                  <MenuItem value="alberta">Alberta</MenuItem>
+                  <MenuItem value="british_columbia">British Columbia</MenuItem>
+                  <MenuItem value="manitoba">Manitoba</MenuItem>
+                  <MenuItem value="new_brunswick">New Brunswick</MenuItem>
+                  <MenuItem value="newfoundland_and_labrador">
+                    Newfoundland and Labrador
+                  </MenuItem>
+                  <MenuItem value="nova_scotia">Nova Scotia</MenuItem>
+                  <MenuItem value="ontario">Ontario</MenuItem>
+                  <MenuItem value="prince_edward_island">
+                    Prince Edward Island
+                  </MenuItem>
+                  <MenuItem value="quebec">Quebec</MenuItem>
+                  <MenuItem value="saskatchewan">Saskatchewan</MenuItem>
+                </Select>
+              </FormControl>
+              <TextField
+                fullWidth
+                sx={{ m: 1, width: "47%" }}
+                id={`secondaryPostalCode${lead.id}`}
+                value={lead.secondaryPostalCode}
+                onChange={(input) =>
+                  handleInputChange(
+                    input.target.value,
+                    lead.id,
+                    "secondaryPostalCode",
+                  )
+                }
+                label="Postal Code"
+              />
+            </>
+          )}
+        </InfoContainer>
+      ))}
+      <ButtonL
+        onClick={handleAddSecondary}
+        style={{ display: showAddSecondaryButton ? "block" : "none" }}
+      >
+        {showAddSecondaryButton
+          ? "+ Add Secondary Address"
+          : "Secondary Address Added"}
```

Comment:
un-necessary -> it will only be visible in the case where the the secondary address has not been added